src/transformers/models/mixtral/configuration_mixtral.py

```diff
@@ -128,7 +128,7 @@ def __init__(
         eos_token_id=2,
         tie_word_embeddings=False,
         rope_theta=1e6,
-        sliding_window=4096,
+        sliding_window=None,
```

Comment:
should we completely remove all logic with respect to `sliding_window` in modeling mixtral?

Reply:
It's not BC

Reply:
So no we can't

Reply:
In any case the mixtral team has already removed the `sliding_window` attribute in the config and it is already breaking as the argument is silently getting ignored in the modeling file: https://github.com/huggingface/transformers/blob/main/src/transformers/modeling_attn_mask_utils.py#L149 / https://github.com/huggingface/transformers/blob/main/src/transformers/models/mixtral/modeling_mixtral.py#L1063

Reply:
We can maybe deprecate the arg through a deprecation cycle

Reply:
As discussed offline, let's keep it for performance BC for users that have pushed a mixtral model with sliding window